lib/signs_ui_web/controllers/auth_controller.ex

```diff
@@ -97,6 +81,27 @@ defmodule SignsUiWeb.AuthController do
     )
   end
 
+  @spec logout_redirect_url_for_provider(Plug.Conn.t(), String.t()) :: String.t()
+  defp logout_redirect_url_for_provider(conn, "cognito") do
```

Comment:
"cognito" here is the provider name from https://github.com/mbta/signs_ui/blob/master/config/prod.exs#L31 yes? Can it look up the strategy `Ueberauth.Strategy.Cognito` from there?

Reply:
It could, but I'm not sure we can do much with that information? Ueberauth strategies don't define a callback for any logout-related stuff so I can't think of any standardized way that we could invoke this across different strategies.